test/REQUIRE

```diff
@@ -1,2 +1,3 @@
-GLPKMathProgInterface 0.4.0 0.4.1
-Ipopt
+GLPKMathProgInterface 0.4 0.5-
+GLPK 0.8 0.9-
+Ipopt 0.4 0.5-
```

Comment:
It's most likely that these tight upper bounds will only create more work for later to bump them because we don't expect breaking changes in the solver packages themselves. Also since this is a test dependency, if by chance something does break it only in the tests and not directly visible to users.